pom.xml

```diff
@@ -53,6 +53,8 @@
         <spring.version>5.2.1.RELEASE</spring.version>
         <retrofit.version>2.5.0</retrofit.version>
         <consul-client.version>1.3.3</consul-client.version>
+        <mongodb.version>3.12.1</mongodb.version>
+        <mongodb-crypt.version>1.0.1</mongodb-crypt.version>
```

Comment:
Could you plz add a comment that these versions should follow Camel and can be removed when [link to an issue] gets fixed? An issue would be nice documenting all the possible improvements.

Reply:
I've added a comment and some links.